src/main/java/org/reaktivity/command/log/internal/spy/OneToOneRingBufferSpy.java

```diff
@@ -62,6 +60,18 @@ public DirectBuffer buffer()
     }
 
     @Override
+    public long producerPosition(DirectBuffer buffer)
+    {
+        return buffer.getLong(buffer.capacity() - TRAILER_LENGTH + TAIL_POSITION_OFFSET);
+    }
+
+    @Override
+    public long consumerPosition(DirectBuffer buffer)
+    {
+        return buffer.getLong(buffer.capacity()  - TRAILER_LENGTH + HEAD_POSITION_OFFSET);
+    }
```

Comment:
No need for `buffer` method parameter, already have the internal `buffer` field.